github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -534,3 +534,44 @@ periodics:
     - name: token
       secret:
         secretName: oauth-token
+- name: periodic-kubevirt-presubmit-requirer
+  cron: "0 1 1 * *"
+  max_concurrency: 1
+  annotations:
+    testgrid-create-test-group: "false"
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+    - org: kubevirt
+      repo: project-infra
+      base_ref: master
+      workdir: true
+  cluster: phx-prow
```

Comment:
We shouldn't run new jobs on phx-prow (preparing PRs for moving all the jobs to prow-workloads)
```suggestion
  cluster: prow-workloads
```